### DISCUSSION TITLE
using --html, only ---pdf does not enable <sup> tag?

### DISCUSSION THREAD

**Discussion by ToruSagami** [marp-team/marp#408] [Q&A] [answered]
Hi, thanks for the project team. Marp is great!

Today I have a question about "sup" html tags. Recently I found the problem (it seemed to not occur several month ago...).

File that reproduces "my problem" is located at:
https://raw.githubusercontent.com/ToruSagami/public_me/main/2023/2023-01-11_convert_sup_and_htmltag_to_pdf.md

Question/Problem is:
Using --html, although all commands except last --pdf did properly work well (transform &lt;font&gt; &lt;ruby&gt; &lt;sup&gt; tags), pdf file that is created by --pdf did not transform &lt;sup&gt; tags. 

```bash
marp --html --preview 2023-01-11_convert_sup_and_htmltag_to_pdf.md
marp --html 2023-01-11_convert_sup_and_htmltag_to_pdf.md
marp --images  --html 2023-01-11_convert_sup_and_htmltag_to_pdf.md marp --pptx  --html 2023-01-11_convert_sup_and_htmltag_to_pdf.md 
marp --pdf  --html 2023-01-11_convert_sup_and_htmltag_to_pdf.md 
```

&lt;sup&gt; tag
NG: .pdf
OK: preview, .html, .png, .pptx.

NG
https://github.com/ToruSagami/public_me/blob/main/2023/2023-01-11_convert_sup_and_htmltag_to_pdf.pdf

OK
https://github.com/ToruSagami/public_me/raw/main/2023/2023-01-11_convert_sup_and_htmltag_to_pdf.pptx

Is this a bug, or needs specical command line option or something?

```
marp -v
@marp-team/marp-cli v2.3.0 (w/ @marp-team/marp-core v3.4.2)
```

**Comment by yhatt** [comment, marked as answer]
It may be a bug from Chrome, similar to https://github.com/marp-team/marp-vscode/issues/393#issuecomment-1364662238. When tried to print the exported HTML to PDF in the browser, `<sup>` has surely vanished.

It affects only when using `default` theme that has the custom style for `<sup>` and `<sub>`, and not affects when using the theme that have no custom styles for them.

Reporting it to https://bugs.chromium.org/p/chromium/issues/ may be worth.

### User-side workaround

```css
sup, sub {
  display: inline-block;
}
```

**Comment by ToruSagami** [comment]
Thanks for your reply.
User-side workaround seems to work well, that is most helpful😀
